Library/Homebrew/keg.rb

```diff
@@ -581,15 +582,28 @@ def normalize_pod2man_outputs!
 
       content = content.gsub(generated_regex, "")
       content = content.lines.map do |line|
-        next line unless line.start_with?(".TH")
-
-        # Split the line by spaces, but preserve quoted strings
-        parts = line.split(/\s(?=(?:[^"]|"[^"]*")*$)/)
-        next line if parts.length != 6
-
-        # pod2man embeds the perl version used into the 5th field of the footer
-        T.must(parts[4]).gsub!(/^"perl v.*"$/, "\"\"")
-        "#{parts.join(" ")}\n"
+        if line.start_with?(".TH")
+          # Split the line by spaces, but preserve quoted strings
+          parts = line.split(/\s(?=(?:[^"]|"[^"]*")*$)/)
+          next line if parts.length != 6
+
+          # man extension remove in man files
+          parts[2]&.gsub!(extension, "")
```

Comment:
For `git-cal` formula, manual suffix doesn't match with macos `.1` and linux `.1p`. So I have moved `.1` to `.1p` manually, or maybe I can add `MAN1EXT=1p` but then the extension cannot be replaced.

man1.1p
```
...
.IX Title "GIT-CAL 1"
...
```
Maybe it is better not to use extension but just use a regex .